attendance/index.html

```diff
@@ -0,0 +1,240 @@
+---
+title: FIRST® Alumni &amp; Mentors Network at Michigan
+---
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    {% include title-styles.html %}
+    <link rel='stylesheet' href='//cdnjs.cloudflare.com/ajax/libs/timepicker/1.3.5/jquery.timepicker.min.css'>
+</head>
+<body>
+    {% include header.html %}
+
+    
+    <form class='w-50 mx-auto attendance-form'>
+        <h1>FAMNM Attendance Form</h1>
+        <div class='form-group my-5'>
+            <label for='meetingType'>What type of meeting?</label>
+            <select class='form-control' id='meetingType'>
+                <option>
+                    ------
+                </option>
+            </select>
+        </div>
+        <div class="my-5">
+            <label for='meetingDate'>Today's Date</label>
+            <input type='date' id='meetingDate'>
+        </div>
+        <div>
+            <label for='meetingStartTime'>Start Time</label>
+            <input class='timepicker' id='meetingStartTime'>
+            <label for='meetingEndTime'>End Time</label>
+            <input class='timepicker' id='meetingEndTime'>
+        </div>
+
+        <div class="my-5">
+            <label for='meetingDescription'>Meeting Description</label>
+            <textarea 
+                maxlength=1000 
+                required=true
+                class='w-100'
+                id='meetingDescription'
+                placeholder='(Optional) A short description of what happened at the meeting.'></textarea>
+        </div>
+
+        <div class="my-5">
+            <label for='uniqname'>Enter uniqnames of people at meeting. Hit enter to add person.</label>
+            <br/>
+            <input type='text' id='uniqname'>
+            <br/>
+            <div id='uniqname-buttons'>
+
+            </div>
+        </div>
+
+        <button type="button" class="btn btn-primary my-3" data-toggle="modal" data-target="#confirmMeeting" id="confirm">Confirm And Submit</button>
+    </form>
+            
+    <!-- Modal -->
+    <div class="modal fade" id="confirmMeeting" tabindex="-1" role="dialog" aria-labelledby="confirmMeetingTitle" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+        <div class="modal-header">
+            <h5 class="modal-title" id="confirmMeetingTitle">Confirm Submission</h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+            </button>
+        </div>
+        <div class="modal-body">
+            <div id="modalError" style="color:red"></div>
```

Comment:
{nit} combine to close "<.../>

Reply:
just tested; this change breaks bc `jquery` shenanigans.